config/config.go

```diff
@@ -18,12 +18,19 @@ func OptionalEnv(varName string, optional string) string {
 }
 
 /*
-	Check for an environment variable value, exit program if not found
+	Check for an environment variable value or the equivalent docker secret, exit program if not found
 */
 func RequiredEnv(varName string) string {
 	envVar := os.Getenv(varName)
-	if envVar == "" {
+	envVarFileName := os.Getenv(varName + "_FILE")
```

Comment:
Instead of adding a "_FILE" suffix, then you have a value, could you detect if the environment value is a valid file path ? For example with the function [os.Stat(path)](https://pkg.go.dev/os#File.Stat).
Like that, you have one less nest in the if statement, so it's more streamlined while still being able to get different logs for direct value vs file value.